src/components/Modal.js

```diff
@@ -55,13 +55,11 @@ class Modal extends React.Component {
   }
 
   _renderFooterContent () {
-    if (this.props.footerContent) {
-      return (
-        <div className='mx-modal-footer-content' style={styles.footerContent}>
-          {this.props.footerContent}
-        </div>
-      );
-    }
+    return (
+      <div className='mx-modal-footer-content' style={styles.footerContent}>
+        {this.props.footerContent ? this.props.footerContent : null}
```

Comment:
haha I don't have to do the check do I.  If's the prop is null already who cares.  Let me try it with out the check.